spec/requests/api/querying_spec.rb

```diff
@@ -94,6 +94,15 @@ def create_vms_by_name(names)
       expect_query_result(:vms, 3, 3)
       expect_result_resources_to_match_hash([{"name" => "cc"}, {"name" => "bb"}, {"name" => "aa"}])
     end
+
+    it "supports case insensitive ordering" do
+      create_vms_by_name %w(Aa aa AA)
+
+      run_get vms_url, :sort_by => "name", :sort_order => "asc", :sort_options => "nocase", :expand => "resources"
```

Comment:
@AparnaKarve yes, except with @Fryguy's suggested `ignore_case` instead of `nocase`